src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './NewCommentForm.scss';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
+import { addPostComment } from '../../api/comments';
 
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
+export const NewCommentForm = ({ postId, setValue, value }) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
 
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+  return (
+    <form
+      className="NewCommentForm"
+      onSubmit={(e) => {
+        e.preventDefault();
+        addPostComment(postId, name, email, body);
+        setName('');
+        setEmail('');
+        setBody('');
+        setValue(current => current + 1);
```

Comment:
current ??
setValue має приймати примітивне значення типу number, а ти передаєш функцію
Якщо потрібно до попереднього значення добавити 1 то потрібно брати це значення і до нього плюсувати 1
```suggestion
        setValue(value + 1);
```